File: affinity/sources/affinity.1.f
! @@name:	affinity.1
! @@type:	F-fixed
! @@operation:	compile
! @@expect:	success
! @@version:	omp_4.0
      PROGRAM EXAMPLE
!$OMP PARALLEL PROC_BIND(SPREAD) NUM_THREADS(4)
      CALL WORK()
!$OMP END PARALLEL
      END PROGRAM EXAMPLE

File: affinity/sources/affinity.1.c
/*
* @@name:	affinity.1
* @@type:	C
* @@operation:	compile
* @@expect:	success
* @@version:	omp_4.0
*/
void work();

int main() 
{

#pragma omp parallel proc_bind(spread) num_threads(4)
   {
      work();
   }

   return 0;

}
